packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -275,12 +279,12 @@ class DeepLinksController extends DisposableController {
           pathErrors:
               _getPathErrorsFromIntentFilterChecks(appLink.intentFilterChecks),
           os: [PlatformOS.android],
-          scheme: [appLink.scheme],
+          scheme: {if (appLink.scheme != null) appLink.scheme!},
```

Comment:
nit: pull `appLink.scheme` into a local so it will be type promoted. We have three null checks for this field currently